lua/lspconfig/pyright.lua

```diff
@@ -7,11 +7,22 @@ if vim.fn.has('win32') == 1 then
   bin_name = bin_name..".cmd"
 end
 
+local root_files = {
+  '.git',
+  'setup.py',
+  'setup.cfg',
+  'pyproject.toml',
+  'requirements.txt',
+}
```

Comment:
I think we should put `.git` at the end of the list since it's least likely to be the correct Python workspace root of the above.